book/src/concepts/system.md

```diff
@@ -66,24 +68,24 @@ Sometimes, it can be useful to get a component in the storage for a specific ent
 
 ```rust,no_run,noplaypen
 # extern crate amethyst;
-# use amethyst::ecs::{System, ReadExpect, WriteStorage};
+# use amethyst::ecs::{Entity, System, WriteStorage};
 # use amethyst::core::Transform;
-# use amethyst::renderer::ActiveCamera;
-struct CameraGoesUp;
+struct WalkPlayerUp {
+    player: Entity,
```

Comment:
This works, but it is a bad practice to store runtime data in systems, as it reduces re-usability.
Would it be possible to have a "MovePlayerUp" component and the system join over (Transform, MoveUp) instead.
MoveUp can have a float inside called move_speed and the system multiplying move_speed by time.delta_seconds() when moving the player.

Reply:
The point is that the reader does not know join at that point.
And no.